src/Akeneo/Connectivity/Connection/back/Infrastructure/Cli/CreateConnectionCommand.php

```diff
@@ -0,0 +1,98 @@
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Akeneo\Connectivity\Connection\Application\Settings\Command\CreateConnectionCommand as CreationCommand;
+
+/**
+ * @author    Willy Mesnage <willy.mesnage@akeneo.com>
+ * @copyright 2020 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class CreateConnectionCommand extends Command
+{
+    protected static $defaultName = 'akeneo:connectivity-connection:create';
+
+    /** @var CreateConnectionHandler */
+    private $createConnection;
+
+    public function __construct(CreateConnectionHandler $createConnection)
+    {
+        parent::__construct();
+        $this->createConnection = $createConnection;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setDescription('Creates a new connection')
+            ->addArgument(
+                'code',
+                InputArgument::REQUIRED,
+                'Code of the connection.'
+            )
+            ->addOption(
+                'flow-type',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                sprintf(
+                    '%s | %s | %s',
+                    FlowType::DATA_SOURCE,
+                    FlowType::DATA_DESTINATION,
+                    FlowType::OTHER
+                ),
+                FlowType::OTHER
+            )
+            ->addOption(
+                'label',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                'Label of the connection. Default will be the provided code.'
+            );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $code = $input->getArgument('code');
+        $label = $input->getOption('label') ?? $code;
+        $flowType = $input->getOption('flow-type');
+
+        try {
+            $command = new CreationCommand($code, $label, $flowType);
+            $connectionWithCredentials = $this->createConnection->handle($command);
+            $output->writeln([
+                '<info>A new connection has been created with the following settings:</info>',
+                sprintf('Code: %s', $connectionWithCredentials->code()),
+                sprintf('Label: %s', $connectionWithCredentials->label()),
+                sprintf('Flow type: %s', $connectionWithCredentials->flowType()),
+                sprintf('Client ID: %s', $connectionWithCredentials->clientId()),
+                sprintf('Secret: %s', $connectionWithCredentials->secret()),
+                '<comment>Make sure to save this password somewhere as we won\'t show it to you twice.</comment>',
+                sprintf('Password: %s', $connectionWithCredentials->password()),
```

Comment:
These informations would be enough: 
```suggestion
                sprintf('Code: %s', $connectionWithCredentials->code()),
                sprintf('Client ID: %s', $connectionWithCredentials->clientId()),
                sprintf('Secret: %s', $connectionWithCredentials->secret()),
                sprintf('Username: %s', $connectionWithCredentials->username()),
                sprintf('Password: %s', $connectionWithCredentials->password()),
                '<comment>Make sure to save this password somewhere as we won\'t be able to show it again.</comment>',
```
Adding not necessary information does not bring value and could be a problem later with BC.